src/views/v2/rsvp/actions.php

```diff
@@ -0,0 +1,37 @@
+<?php
+/**
+ * Block: RSVP
+ * Actions
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/actions.php
+ *
+ * See more documentation about our Blocks Editor templating system.
+ *
+ * @link {INSERT_ARTICLE_LINK_HERE}
+ *
+ * @since TBD
+ * @version TBD
+ */
+
+$step = ! empty( $_GET[ 'step' ] ) ? sanitize_text_field( $_GET[ 'step' ] ) : '';
+?>
```

Comment:
* :warning: Processing form data without nonce verification.
* :no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.
* :warning: Processing form data without nonce verification.
* :no_entry_sign: Array keys must NOT be surrounded by spaces if they only contain a string or an integer.

<sub>hash: 41e7b2b96ea3f51a91fcb7d22b4e1655</sub>